src/directives/ctr-input.ts

```diff
@@ -99,7 +100,9 @@ export class CtrInput {
             if (this.completer.hasHighlighted()) {
                 event.preventDefault();
             }
-            this.handleSelection();
+            if (!this.forceSelection || this.completer._hasHighlighted) {
```

Comment:
I do not remember precisely what I was trying to do with the TAB, I guess maybe leaving the menu opened on TAB but this would prevent from switching to a next input in the form, which is pretty much what should happen. Otherwise you need to tqke your mouse to leave the field, and would be some kind of autofocus on TAB.  

So I guess only ENTER key should be concerned.